recipes/cytoolz/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.7.5" %}
+
+package:
+  name: cytoolz
+  version: {{ version }}
+
+source:
+  fn: cytoolz-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/cytoolz/cytoolz-{{ version }}.tar.gz
+  md5: 15fbc1c56fca80cbca0e89f37c93d432
+
+build:
+  number: 0
+  script: python setup.py install --record=record.txt
```

Comment:
As best as possible, but not always.  `cytoolz.__toolz_version__` declares the `toolz` API covered by `cytoolz`.

For the purpose of relationship dependency, `'toolz >= 0.8.0'` (as declared https://github.com/pytoolz/cytoolz/blob/master/setup.py#L110) should be sufficient.  The next version of both `toolz` and `cytoolz` will be `0.8.0`.  Working on releasing now; I'm just double-checking some things.